src/vendor/wordpress/_wordpress.scss

```
@@ -111,6 +106,47 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  /**
+   * 1. Setting a width of `100%` prevents resized and centered images from
+   *    breaking layout when placed after a right-aligned image in a post.
+   */
+  &.aligncenter {
+    width: 100%; /* 1 */
+  }
+
+  &.alignfull {
+    margin-left: calc(-50vw + 50%);
+    margin-right: calc(-50vw + 50%);
+  }
+
+  /*
+   * 1. Applies an opinionated wide-image width that allows an image element to
+   *    spill slightly outsides its container at larger viewports.
+   */
+  &.alignwide {
+    margin-left: calc(-50vw + 50%);
+    margin-right: calc(-50vw + 50%);
+    @media (min-width: breakpoint.$l) {
+      left: calc(12vw / -2); /* 1 */
+      margin-left: auto;
+      margin-right: auto;
+      width: calc(100% + 12vw); /* 1 */
```

Would this accomplish the same thing?

```suggestion
      margin-left: -6vw; /* 1 */
      margin-right: -6vw; /* 1 */
```

---

It does! I am introducing these changes manually.